app/controllers/settings/admission_webhook_controller.rb

```diff
@@ -0,0 +1,67 @@
+# Settings::AdmissionWebhook is responsibe to manage all the requests
+# related to the kubernetes auditing feature.
+class Settings::AdmissionWebhookController < SettingsController
+  def index
+    @form = AdmissionWebhookForm.new
+    set_instance_variables
+  end
+
+  def create
+    @form = AdmissionWebhookForm.new(admission_webhook_params)
+    @errors = Pillar.apply(
+      admission_webhook_pillars,
+      unprotected_pillars: admission_webhook_pillars.keys
+    )
+    if @form.valid? && @errors.empty?
+      redirect_to settings_admission_webhook_index_path,
+        notice: "Admission webhook settings successfully saved."
+    else
+      set_instance_variables
+      render action: :index, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def set_instance_variables
+    @admission_webhook_enabled = Pillar.value(pillar: :api_admission_webhook_enabled) == "true"
+    @certificate = Pillar.value(pillar: :api_admission_webhook_cert)
+    @key = Pillar.value(pillar: :api_admission_webhook_key)
+  end
+
+  def admission_webhook_params
```

Comment:
can you still make sure to filter the cert/key from the logs in `config/initializers/filter_parameter_logging.rb` ?

Reply:
I thought about that but for file inputs the content doesn't appear as for text inputs. I can paste the content here if you want. 

If we still think that it's necessary to filter it, I can add it.

Reply:
hmm in my case it appears, but it's the devenv. if it doesnt appear in production it's totally fine ;)

Reply:
Oh. You mean the `current_cert` and `current_key`. Yeah, you are right. I thought that it was the `key` and `certificate` inputs. :sweat_smile:

Reply:
Fixed!